Library/Homebrew/formula.rb

```diff
@@ -2142,6 +2142,7 @@ def stage
         stage_env[:_JAVA_OPTIONS] =
           "#{ENV["_JAVA_OPTIONS"]&.+(" ")}-Duser.home=#{HOMEBREW_CACHE}/java_cache"
         stage_env[:GOCACHE] = "#{HOMEBREW_CACHE}/go_cache"
+        stage_env[:GOPATH] = "#{HOMEBREW_CACHE}/go_mod_cache"
```

Comment:
Worth setting in `run_test` too or is it basically never going to be used there?

Reply:
Umm, I don't think we run any `go` command often in test blocks.

Reply:
np, shipit!